nightly-playground/scripts/search_relevance.sh

```diff
@@ -0,0 +1,404 @@
+#!/bin/sh
+
+# This script sets up sample data to exercise the features of the Search Relevance Workbench.
+#
+# ./search-relevance.sh --url https://${{needs.validate-and-deploy.outputs.ENDPOINT}}/${{needs.validate-and-deploy.outputs.PLAYGROUND_ID}}
+#
+# It will clear out any existing indexes as part of running it.
+
+# Helper script for capturing return values from curl commands
+exe() { (set -x ; "$@") | jq | tee RES; echo; }
+
+
+# Default URL if not provided
+OPENSEARCH_SERVER_URL=${OPENSEARCH_SERVER_URL:-"$OPENSEARCH_SERVER_URL"}
+
+# Allow command-line override
+if [ "$1" = "--url" ] && [ -n "$2" ]; then
+  OPENSEARCH_SERVER_URL="$2"
+  shift 2
+fi
+
+echo "Using OpenSearch server: $OPENSEARCH_SERVER_URL"
+
+echo Deleting ecommerce sample data
+(curl -s -X DELETE "$OPENSEARCH_SERVER_URL/ecommerce" > /dev/null) || true
+
+ECOMMERCE_DATA_FILE="esci_us_opensearch-2025-06-06.json"
+# Check if data file exists locally, if not download it
+if [ ! -f "$ECOMMERCE_DATA_FILE" ]; then
+  echo "Data file not found locally. Downloading from S3..."
+  wget https://o19s-public-datasets.s3.amazonaws.com/esci_us_opensearch-2025-06-06.json
```

Comment:
Can we commit this file into the repo? It is recommended to have the data reviewed rather than downloading on-the go.

Reply:
we totally can....    Unzipped it's 230 MB, but if I gzip it, it shrinks to 63mb.  Is that too large?  We *could* also look at the file and see if we can shrink it, we don't need ALL those products for our sample data...  Any thoughs on waht a size target for checking in would be?

Reply:
Github limits: https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#file-size-limits

Reply:
I see a valid concern here, @epugh to begin with can we have a slim dataset that can be added to GitHub (that would still be a useful one) and use that to index to the cluster?

Reply:
Do you have a sense of the target size?   What are we comfortable with checking in?   I need to make sure the product data set and the judgement dataset overlap reasonably as well.

Reply:
FYI today there is a job https://build.ci.opensearch.org/job/ml-models-release (related jenkinsfile link https://github.com/opensearch-project/opensearch-py-ml/blob/main/jenkins/ml-models.JenkinsFile) that uploads models to OpenSearch s3 buckets securely. If required we can implement a similar solution. 

But I would opt to add in to GitHub as much as possible as its a demo data.

Reply:
WDYT can we have a script that can generate data during runtime and index? We can keep this as an option for future datasets.